releaseNotes/SpringCloudAzureAppConfigurationConfig.md

```diff
@@ -2,6 +2,50 @@
 
 [Source code][source_code] | [Package (Maven)][package] | [Product documentation][docs] | [Samples][samples]
 
+## 1.1.0.M6 - December 09, 2019
+
+* Updated support for AAD. System Assigned identity previously required; config store name, and the Contributor role. Now it requires; config store config store name, [client id][client_id], and the App Configuration Data Reader role.
+
+```properties
+spring.cloud.azure.appconfiguration.managed-identity.client-id=[client id]
+spring.cloud.azure.appconfiguration.stores[0].name=[config store name]
+```
+
```

Comment:
How about this?

* Updated managed identity support for both system-assigned managed identity and user-assigned managed identity.

  **Before**
The managed identity is assigned *Contributor* role to the App Configuration instance.

  **After**
  - The system-assigned managed identity also requires the client-id to be set in addition to the config store name, for example,
    ```properties
    spring.cloud.azure.appconfiguration.managed-identity.client-id=[client id]
    spring.cloud.azure.appconfiguration.stores[0].name=[config store name]
    ```
  - The managed identity should be assigned *App Configuration Data Reader* role to the App Configuration instance.